src/PrestaShopBundle/Form/Admin/Product/ProductCombinationBulk.php

```diff
@@ -100,9 +100,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => $this->translator->trans('Low stock level', array(), 'Admin.Catalog.Feature'),
                 'attr' => array(
                     'placeholder' => $this->translator->trans(
-                        'If empty, it keeps the original value',
+                        'This field can only add a value',
```

Comment:
I think we should have a tooltip here instead, saying something like "You can only increase or decrease low stock levels in bulk, but you cannot disable it. To disable a low stock level, do it combination by combination."